sql/1.107/github_commit_comments_table.sql

```diff
@@ -0,0 +1,18 @@
+BEGIN;
+
+CREATE TABLE "github_commit_comment" (
+    "id" serial NOT NULL,
+    "sha" bigint NOT NULL REFERENCES github_commit(id) ON DELETE CASCADE,
+    "github_repo_id" bigint NOT NULL REFERENCES github_repo(id) ON DELETE CASCADE,
+    "github_user_id" bigint NOT NULL REFERENCES github_user(id) ON DELETE CASCADE,
+    "position" bigint NOT NULL,
+    "line" bigint NOT NULL,
+    "number" bigint NOT NULL,
+    "body" text NOT NULL,
+    "created_at" timestamp with time zone NOT NULL,
+    "gh_data" text NOT NULL,
+    
+    PRIMARY KEY (commit_id, number)
```

Comment:
@zachthompson there are a few more errors relating to commit comments i'm seeing like in the 
`update_repo_commit_comment_from_data` in `GitHub.pm`. These are all relating to me thinking through how to identify commit comments unique constraints. I think we should be using the `sha` of the comment's commit and the `commit_id` which would be unique to that comment within the commit, if that makes sense...I need to remove the `number` key and replace with `commit_id` in a few places. Here's the API reference if you're interested - https://developer.github.com/v3/repos/comments/#list-commit-comments-for-a-repository